boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig

```diff
@@ -2,6 +2,8 @@ CONFIG_SOC_QEMU_CORTEX_A53=y
 CONFIG_BOARD_QEMU_CORTEX_A53=y
 CONFIG_ARM_ARCH_TIMER=y
 CONFIG_QEMU_ICOUNT=n
```

Comment:
Do we want `CONFIG_QEMU_ICOUNT=y` here as well?

Reply:
Oh, this was just fixed.

Reply:
Sorry, I forgot a note.  Yes, the smp platform was still showing failures so I removed the icount setting there; it's just voodoo anyway, SMP in qemu is always implemented with host threads, so it's always subject to host timer skew regardless of how the emulator works internally.